src/platform/site-wide/component-library-analytics-setup.js

```diff
@@ -4,22 +4,9 @@
  * to translate component library actions into analytics dataLayer events.
  */
 import _recordEvent from 'platform/monitoring/record-event';
-import { getSectionLabel } from 'applications/static-pages/subscription-creators/subscribeAccordionEvents';
 
 const analyticsEvents = {
   Modal: [{ action: 'show', event: 'int-modal-show', prefix: 'modal' }],
-  'va-accordion': [
-    {
-      action: 'expand',
```

Comment:
I don't think any of the web component analytics details should be removed, should they?

Reply:
This is for the V1 accordion. The V3 accordion doesn't track "expand" and "collapse", it instead tracks "VaAccordionToggled", so new analytics will need to be added to support that. I plan on making a separate ticket to track that work.

Reply:
But I see `expand` and `collapse` analytics actions in the current accordion component:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-accordion/va-accordion.tsx#L130

Reply:
Oh! You're right, they are still there. I was only looking at what was being emitted from va-accordion-item. I will update this.